spec/classes/net_process_log_spec.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'profile_audit::net_process_log' do
+  on_supported_os.each do |os, os_facts|
+    context "on #{os}" do
+      let(:facts) { os_facts }
+
+      it { is_expected.to compile }
+    end
+  end
+end
```

Comment:
Why is this here?

I don't see any references to this in the Puppet classes.

Reply:
I used `pdk new class net_process_log_spec` to create the class, which also creates that file. We talk about it a bit here : https://wiki.ncsa.illinois.edu/display/PUPPET/PDK+-+Puppet+Development+Kit

Not any mentions in the wiki about what/how the file is used. I'm not really sure myself but looks like it's used for unit testing. We don't really do any unit testing but was just going to leave it in since it wouldn't hurt, but I can remove it if we don't want extra clutter.

Reply:
Ben is correct, it is an autogenerated file from PDK for unit testing.
If reviewing puppet code on your local machine, you can ignore PDK autogenerated files. See the section "Ignore PDK auto-generated files during a git diff" on the same page linked above.

Reply:
I was confused. I missed that it was a `spec` file. I was mistakenly thinking this was a custom fact. 
So, nevermind on this.